geode-core/src/main/java/org/apache/geode/internal/cache/persistence/PersistenceInitialImageAdvisor.java

```diff
@@ -107,6 +107,7 @@ public InitialImageAdvice getAdvice(InitialImageAdvice previousAdvice) {
                   shortDiskStoreID, regionPath, advice.getReplicates());
             }
             if (persistenceAdvisor.acquireTieLock()) {
+              // TODO: Pass current advice here?
```

Comment:
Remove TODO? It may imply that there is work to do...

Reply:
done